stable/optimus/values.yaml

```diff
@@ -84,11 +89,15 @@ service:
   type: ClusterIP
   ports:
     - port: 80
-      targetPort: optimus
+      targetPort: http
       protocol: TCP
       name: http
+    - port: 8081
+      name: grpc
+      targetPort: grpc
+      protocol: TCP
   annotations:
-    projectcontour.io/upstream-protocol.h2c: 80,http
+    projectcontour.io/upstream-protocol.h2c: grpc
 
 ingress:
```

Comment:
could you add `additionalIngress` related fields on values.yaml?

Reply:
please refer to this https://github.com/goto/charts/blob/main/stable/app/values.yaml